### PR TITLE
[wip] Handle conversion with svg-to-img package

### PR DESCRIPTION
**Pull request by oa495**

### CODE REVIEW

File: packages/@icon-magic/generate/src/plugins/svg-to-raster.ts [hunk 1]
```diff
@@ -104,19 +104,18 @@ async function generatePng(
...
+ * @param outputPath path to write the webp file after it's created
  */
-function convertToWebp(pathToPng: string, outputPath: string): Promise<string> {
-  return new Promise<string>((resolve, reject) => {
-    webp.cwebp(pathToPng, outputPath, '-q 80', function(status: string) {
-      !!~status.indexOf('100') ? resolve(outputPath) : reject();
-    });
-  });
-}
+async function generateWebp(
```
Comment (ramitha) [inline packages/@icon-magic/generate/src/plugins/svg-to-raster.ts]: We don't need to create a wrapper around this. We can call it directly on line 65.

File: packages/@icon-magic/svg-to-png/src/index.ts [hunk 2]
```diff
@@ -37,3 +38,37 @@ export async function convert(contents: string, options: SVGToPNGOptions): Promi
     });
   });
 }
+
```
Comment (ramitha) [inline packages/@icon-magic/svg-to-png/src/index.ts]: I'm wondering if we need this entire package at all. We could just have svg-to-img be a dependency of generate. Let's figure this out after we run the test the output sizes.

File: packages/@icon-magic/svg-to-png/src/index.ts [hunk 3]
```diff
@@ -37,3 +38,37 @@ export async function convert(contents: string, options: SVGToPNGOptions): Promi
...
+/**
+ * Converts a svg file to webp and writes to the outputPath
+ * @param input path to the input file in .png format
+ * @param options options for the conversion
+ * @param outputPath path where the output needs to go
+ */
+export async function convertToPng(contents: string, options: SVGToPNGOptions, outputPath: string): Promise<string | Buffer> {
+  if (!contents) {
+    throw new Error("No contents discovered.");
```
Comment (ramitha) [inline packages/@icon-magic/svg-to-png/src/index.ts]: Use the new Logger :)

File: packages/@icon-magic/svg-to-png/src/index.ts [hunk 4]
```diff
@@ -37,3 +38,37 @@ export async function convert(contents: string, options: SVGToPNGOptions): Promi
...
+}
+
+/**
+ * Converts a svg file to webp and writes to the outputPath
+ * @param input path to the input file in .png format
+ * @param outputPath path where the output needs to go
+ */
+export async function convertToWebp(contents: string, outputPath: string): Promise<string | Buffer> {
+  if (!contents) {
+    throw new Error("No contents discovered.");
```
Comment (ramitha) [inline packages/@icon-magic/svg-to-png/src/index.ts]: Likewise.

File: packages/@icon-magic/svg-to-png/src/index.ts (hunk 2 above)
Comment (oa495) [reply packages/@icon-magic/svg-to-png/src/index.ts]: yep I agree, it would just be one file with ~ 25 lines since we're removing the puppeteer stuff.

**Pull request closed without merging**